api/security/api-key-authorize.test.js

```diff
@@ -220,4 +220,35 @@ describe('Api Key Authorizer', () => {
       Authorizer(request, response, next)
     })
   })
+
+  describe('has super admin API key', () => {
+    test('should succeed on GET request', done => {
+      const apiKey = {
+        api_key: 'some-api-key',
+        api_pattern: [
+          '.*'
+        ],
+        type: 'super'
+      }
+
+      const requestHeaders = {}
+      requestHeaders['x-api-key'] = apiKey.api_key
+
+      const request = {
+        method: 'GET',
+        query: {
```

Comment:
With this here, there's no way to distinguish in this file that this test is doing anything different from the valid API key test above. Perhaps we just need to add another test without a query parameter.